app/utils/file/index.ts

```diff
@@ -267,9 +267,14 @@ export const isImage = (file?: FileInfo | FileModel) => {
         return false;
     }
 
-    const mimeType = 'mime_type' in file ? file.mime_type : file.mimeType;
+    let mimeType = 'mime_type' in file ? file.mime_type : file.mimeType;
+    if (!mimeType && file.extension) {
+        mimeType = lookupMimeType(file.extension);
+    } else if (!mimeType && file.name) {
+        mimeType = lookupMimeType(file.name);
+    }
```

Comment:
```suggestion
    if (isGif(file)) {
        return true;
    }

    let mimeType = 'mime_type' in file ? file.mime_type : file.mimeType;
    if (!mimeType) {
        mimeType = lookupMimeType(file.extension) || lookupMimeType(file.name);
    }

    return (mimeType?.startsWith('image/'));
```

Reply:
0/5 for a small refactor.  
- return ealy if file is a gif
- remove else block